rpmalloc/rpmalloc.c

```diff
@@ -548,6 +548,12 @@ static size_t os_page_size;
 #if defined(_MSC_VER) && !defined(__clang__)
 #define TLS_MODEL
 #define _Thread_local __declspec(thread)
+#elif defined(__ANDROID__)
+#if __ANDROID_API__ >= 29 && defined(__NDK_MAJOR__) && __NDK_MAJOR__ >= 26
+    #define TLS_MODEL __attribute__((tls_model("local-dynamic")))
+#else
+    #define TLS_MODEL
+#endif
 #else
 // #define TLS_MODEL __attribute__((tls_model("initial-exec")))
```

Comment:
Just noticing that this was probably [temporarily commented out](https://github.com/mjansson/rpmalloc/commit/955f44b6c35662e19c3e5f8ae93d8b658134c2ca) to test Android, but it has fortunately already been restored in 20844207cff008329303f07c1ae940bacceeb120.